src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -398,12 +405,16 @@ public function deleteMenuThumbnailAction(Request $request, $categoryId, $menuTh
     /**
      * Toggle category status.
      *
-     * @param Request $request
+     * @AdminSecurity(
+     *     "is_granted(['update', 'delete'], request.get('_legacy_controller'))",
```

Comment:
```suggestion
     *     "is_granted(['update'], request.get('_legacy_controller'))",
```
This controller does not allow to delete a Category 🤔?

Reply:
Oh, I see, it's how it has been written. Mmmm ... wdyt ? I think we can remove the Delete here.

Reply:
Done.